optuna/integration/_lightgbm_tuner/alias.py

```diff
@@ -53,6 +53,10 @@ def _handling_alias_parameters(lgbm_params: Dict[str, Any]) -> None:
         ],
     },
     {"metric_name": "map", "alias_names": ["mean_average_precision"]},
+    {
+        "metric_name": "l2",
+        "alias_names": ["regression", "regression_l2", "l2", "mean_squared_error", "mse"],
+    },
```

Comment:
Could you add a test case for this case in `tests/integration_tests/lightgbm_tuner_tests/test_alias.py`?
It'll be something like this:

```python
    for alias in ["regression", "regression_l2", "l2", "mean_squared_error", "mse"]:
        lgbm_params = {"metric": alias}
        _handling_alias_metrics(lgbm_params)
        assert lgbm_params["metric"] == "l2"
```

Reply:
Will do.